platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/widgets/MyLocationView.java

```diff
@@ -423,22 +421,18 @@ public void onRestoreInstanceState(Parcelable state) {
   private void toggleGps(boolean enableGps) {
     LocationEngine locationEngine = LocationSource.getLocationEngine(getContext());
     if (enableGps) {
-      // Set an initial location if one available
-      Location lastLocation = locationEngine.getLastLocation();
```

Comment:
can you clarify manual update? Expose a setter for end-users? or saving the last location ourselves? some background on why these lines were removed, this line will always return null as the LocationEngine hasn't been connected (this is due the change in LOST 1.0 -> 2.0).